geolocator/ios/Classes/Handlers/LocationAccuracyHandler.h

```diff
@@ -17,7 +17,9 @@ typedef enum {
 
 @interface LocationAccuracyHandler : NSObject
 
-- (void) getLocationAccuracyWithResult:(FlutterResult)result;
+- (void) getLocationAccuracyWithResult:(FlutterResult _Nonnull)result;
+- (void) requestTemporaryFullAccuracyWithResult:(FlutterResult _Nonnull)result
+                                      arguments:(id _Nullable)arguments;
```

Comment:
The `LocationAccuracyHandler` should simply define the parameters it needs and not accept something that needs to be parsed first:

```suggestion
                                      purpuseKey:(NSString *)purposeKey;
```